### PR TITLE
Kill any orphaned tests

### PR DESCRIPTION
**Pull request by scotthain**
Signed-off-by: Scott Hain <shain@chef.io>

### Description

[Please describe what this change achieves]

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

### Check List

- [ ] New functionality includes tests
- [ ] All tests pass
- [ ] RELEASE\_NOTES.md, has been updated if required (not required for bugfixes, required for API changes)
- [ ] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

**Comment by scotthain** [comment]
@chef/jex-team

File: ci/verify-chef.sh [hunk 1]
```diff
@@ -8,6 +8,9 @@ export TMPDIR
 sudo rm -rf $TMPDIR
 mkdir -p $TMPDIR
 
+# Verify that we kill any orphaned test processes. Kill any orphaned rspec processes.
+ps ax | egrep 'rspec' | grep -v grep | awk '{ print $1 }' | xargs sudo kill -s TERM || true
```
Comment (lamont-granquist) [inline ci/verify-chef.sh]: `ps ax` make solaris or aix angry?

also you should just SIGKILL and nuke from orbit.  it's the only way to be sure...

Comment (scotthain) [reply ci/verify-chef.sh]: Done

Comment (scotthain) [reply ci/verify-chef.sh]: Also `ps ax` works on sol and aix, the flags with `-` sometime cause issues.

**Comment by lamont-granquist** [comment]
@scotthain sysv-vs-bsd ps options, or is that the point of the `|| true` that its more important on linuxen and you don't care about slowaris?

**Comment by scotthain** [comment]
@lamont-granquist the point of the `|| true` is it will exit non-0 if there are no leftovers (and kill has nothing to kill). This is just to ensure that it continues on the happy path.

**Comment by lamont-granquist** [comment]
so....   `ps -ef`?

**Comment by lamont-granquist** [comment]
well okay then, everyone has gotten on the bsd ship...

**Pull request merged**